exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -0,0 +1,1480 @@
+	orders := []order.Detail{}
+	switch getOrdersRequest.AssetType {
+	case asset.Futures:
+		if len(getOrdersRequest.Pairs) == 1 {
+			pair = format.Format(getOrdersRequest.Pairs[0])
+		}
+		sideString, err := ku.orderSideString(getOrdersRequest.Side)
+		if err != nil {
+			return nil, err
+		}
+		oType, err := orderTypeToString(getOrdersRequest.Type)
+		if err != nil {
+			return nil, err
+		}
+		futuresOrders, err := ku.GetFuturesOrders(ctx, "active", pair, sideString, oType, getOrdersRequest.StartTime, getOrdersRequest.EndTime)
+		if err != nil {
+			return nil, err
+		}
+		var enabledPairs currency.Pairs
+		enabledPairs, err = ku.GetEnabledPairs(asset.Futures)
+		if err != nil {
+			return nil, err
+		}
+		for x := range futuresOrders.Items {
+			if !futuresOrders.Items[x].IsActive {
+				continue
+			}
+			dPair, err := enabledPairs.DeriveFrom(futuresOrders.Items[x].Symbol)
+			if err != nil {
+				return nil, err
+			}
+			for i := range getOrdersRequest.Pairs {
+				if !getOrdersRequest.Pairs[i].Equal(dPair) {
+					continue
+				}
+				side, err := order.StringToOrderSide(futuresOrders.Items[x].Side)
+				if err != nil {
+					return nil, err
+				}
+				oType, err := order.StringToOrderType(futuresOrders.Items[x].OrderType)
+				if err != nil {
+					return nil, fmt.Errorf("asset type: %v order type: %v err: %w", getOrdersRequest.AssetType, getOrdersRequest.Type, err)
+				}
+				orders = append(orders, order.Detail{
+					OrderID:         futuresOrders.Items[x].ID,
+					Amount:          futuresOrders.Items[x].Size,
+					RemainingAmount: futuresOrders.Items[x].Size - futuresOrders.Items[x].FilledSize,
+					ExecutedAmount:  futuresOrders.Items[x].FilledSize,
+					Exchange:        ku.Name,
+					Date:            futuresOrders.Items[x].CreatedAt.Time(),
+					LastUpdated:     futuresOrders.Items[x].UpdatedAt.Time(),
+					Price:           futuresOrders.Items[x].Price,
+					Side:            side,
+					Type:            oType,
+					Pair:            dPair,
+				})
+			}
+		}
+	case asset.Spot, asset.Margin:
+		if len(getOrdersRequest.Pairs) == 1 {
+			pair = format.Format(getOrdersRequest.Pairs[0])
+		}
+		sideString, err := ku.orderSideString(getOrdersRequest.Side)
+		if err != nil {
+			return nil, err
+		}
+		oType, err := ku.orderTypeToString(getOrdersRequest.Type)
+		if err != nil {
+			return nil, fmt.Errorf("asset type: %v order type: %v err: %w", getOrdersRequest.AssetType, getOrdersRequest.Type, err)
+		}
+		spotOrders, err := ku.ListOrders(ctx, "active", pair, sideString, oType, "", getOrdersRequest.StartTime, getOrdersRequest.EndTime)
+		if err != nil {
+			return nil, err
+		}
+		var enabledPairs currency.Pairs
+		enabledPairs, err = ku.GetEnabledPairs(asset.Futures)
+		if err != nil {
+			return nil, err
+		}
+		for x := range spotOrders.Items {
+			if !spotOrders.Items[x].IsActive {
+				continue
+			}
+			dPair, err := enabledPairs.DeriveFrom(spotOrders.Items[x].Symbol)
+			if err != nil {
+				return nil, err
+			}
```

Comment:
```suggestion
if err !=nil {
    if errors.Is(err, currency.ErrPairNotFound) {
        continue
    }
    return nil, err
}
```

Just a bit nicer for the user